packages/block-library/src/group/editor.scss

```diff
@@ -8,6 +8,12 @@
 		left: 0;
 		right: 0;
 	}
+
+	user-select: none;
+
+	> * {
+		user-select: all;
+	}
```

Comment:
What happens if a group is within a group, which rule should apply? I assume the "none" but It seems this won't work for it.

Reply:
Oh yes, I guess `.group > *` is stronger than `.group`.

Reply:
Also I guess it would be good to have a general solution, not just for groups, but it feels a bit dangerous to disable selection for all non editable content.

Reply:
yea, maybe we can start with groups. The other thing is should this be `user-select: all;` or `user-select: reset;` or something (we don't know what's inside the group, so would be good to have a way to reset to browser default.